frontend/packages/dev-console/src/components/guided-tour/GuidedTourText.tsx

```diff
@@ -20,7 +20,7 @@ export const devPerspectiveTourText = <DevPerspectiveTourText />;
 
 export const perspectiveSwitcherTourText = (
   <>
-    <p>Switch between the Developer and Admin perspectives.</p>
+    <p>Switch between the Developer and Administrator perspectives.</p>
```

Comment:
Shall we make similar change below as well `Use the Admin perspective` to `Use the Administrator perspective`